wiremock/mappings/060-dps-frontend-footer-component-mapping.json

```diff
@@ -0,0 +1,19 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPattern": "/footer"
+  },
+  "response": {
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "status": 200,
+    "jsonBody": [
```

Comment:
This will need changing so its not an array (so that it works when we run it locally 👍 )